scripts/release/mirror.js

```diff
@@ -12,13 +12,49 @@ const { browsers } = bcd;
  * @typedef {import('../types').ReleaseStatement} ReleaseStatement
  */
 
+const matchingSafariVersions = new Map([
+  ['≤4', '≤3'],
+  ['1', '1'],
+  ['1.1', '1'],
+  ['1.2', '1'],
+  ['1.3', '1'],
+  ['2', '1'],
+  ['3', '1'],
```

Comment:
```suggestion
  ['3', '2'],
```

I think? Safari 3 for macOS was WebKit 523.10 and iOS 1.1.3 was WebKit 420.1 and iOS 2.0 525.18.1.

Reply:
I've carried this along from https://gist.github.com/foolip/6872cf5350dccf6224fce984fb73fba1 and the UA string in https://github.com/mdn/browser-compat-data/pull/11173#issuecomment-865201634 was:

> Mozilla/5.0 (iPhone; U; CPU iPhone OS 2_0 like Mac OS X; en-us) AppleWebKit/525.18.1 (KHTML, like Gecko) Version/3.1.1 Mobile/5A347 Safari/525.20

How to interpret that "Version/3.1.1" I don't know, but going by the WebKit version you're right.

Reply:
Historically the Safari version string was separately coded for iOS and macOS, and it was likely largely an approximate decision about rough correlations (and there's one or two cases later where the iOS Safari version stayed on the `.0` major version even after trunk WebKit releases!), especially given it was (IIRC) only exposed in the UA string.

I suspect it was decided to call it `3.1.1` as it was closer to that macOS Safari release than previous ones, even though it appears to have branched later than the macOS release.